notebooks/qg/hybridqg_inference.ipynb

```diff
@@ -0,0 +1,377 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "489bc0df",
+   "metadata": {},
+   "source": [
+    "# Multihop Hybrid (Table + text) Question Generation: Inference example\n",
+    "\n",
+    "In this notebook, we will show how to use a pretrained HybridQG model to generate questions. We first sample hybrid chains from a given table and its linked passages, and then use a text-to-text transformer (T5) to transcribe the chains to a natural language question. "
```

Comment:
Can you show intermediate output in the notebook, for example what the chains look like?

Reply:
I have added a code snippet for printing hybrid chains.